### PR TITLE
test: fix failing runtime-workflow integration tests

### PR DESCRIPTION
**Pull request by machi1990**
Fix the issue by making sure that there is no shared context between each test.

Previous after the first test is executed, subsequent tests were failing with the following message:
```
TypeError: Cannot redefine property: Symbol(apolloServerExecutionDispatcherWillResolveField)

        at Function.defineProperty (<anonymous>)

      at Object.<anonymous> (../node_modules/apollo-server-core/src/requestPipeline.ts:431:14)

      at fulfilled (../node_modules/apollo-server-core/dist/requestPipeline.js:5:58)
 ```
While looking at the code, it seemed to me like the context we passed to apollo server was a global one and hence between multiple tests which resulted in a failure as we attempted to redifine a property that already was defined in the previous ran.

### CODE REVIEW

**Comment by machi1990** [comment]
Opening as draft to see what CI will say and we can chat about the approach.

File: integration/tests/runtime-workflow.ts [hunk 1]
```diff
@@ -45,28 +50,31 @@ beforeAll(async () => {
...
     console.log(e);
     throw e;
   }
 })
 
+beforeEach(() => {
+  const services = createKnexPGCRUDRuntimeServices(runtimeEngine.getDataSourceModels(), modelSchema, db, new PubSub());
+  const runtime = runtimeEngine.buildRuntime(services);
+  server = new ApolloServer({
+    typeDefs: printSchema(runtime.schema),
```
Comment (craicoverflow) [inline integration/tests/runtime-workflow.ts:68]: `printSchema` actually removes directives from the schema, so if our tests have them in future this won't validate them.

You could replace this with

```suggestion
schema: runtime.schema,
```

I haven't tested the implications of this, but it should work. If we still need to use typeDefs you can use this `printSchemaWithDirectives`:
 https://github.com/aerogear/graphback/blob/2c72ddb03058a6b5c478fe28067325aebd172d5e/packages/graphback-core/src/utils/printSchemaWithDirectives.ts#L4

File: integration/tests/runtime-workflow.ts [hunk 2]
```diff
@@ -130,13 +138,18 @@ const getConfig = async () => {
...
+  }
+
+  const loadedDocuments = await loadDocuments(path.resolve(`./output/client/**/${name}.graphql`), {
     loaders: [
       new GraphQLFileLoader()
     ]
   });
 
-  return documents[0];
+  documentsCache[name] = loadedDocuments[0];
```
Comment (craicoverflow) [inline integration/tests/runtime-workflow.ts:151]: :+1:

**Comment by wtrocki** [comment]
Amazing work @machi1990 +1 from me!

File: integration/tests/runtime-workflow.ts (hunk 1 above)
Comment (machi1990) [reply integration/tests/runtime-workflow.ts]: I did not change the way schema are passed around, but it is a solid suggestion which I have applied, let's see what CI will say :-)

**Pull request merged**